artichoke-backend/src/extn/core/kernel/require.rs

```diff
@@ -281,8 +281,8 @@ impl RelativePath {
     }
 }
 
-fn load_error(interp: &Artichoke, filename: &[u8]) -> Result<LoadError, Exception> {
-    let mut message = String::from("cannot load such file -- ");
-    string::format_unicode_debug_into(&mut message, filename)?;
-    Ok(LoadError::new(interp, message))
+fn load_error(interp: &Artichoke, filename: &[u8]) -> LoadError {
+    let mut message = String::from("cannot load such file -- ").into_bytes();
+    message.extend_from_slice(filename);
+    LoadError::new_raw(&interp, message)
```

Comment:
The extra `&` is not needed on the `interp`. `LoadError::new_raw` takes an `&Artichoke`, which `interp` already is. This extra reference gets us a `&&Artichoke` which is immediately dereferenced by the compiler to get us back the `&Artichoke` (this is called auto-deref).

There is no runtime impact, but can make the code trickier to read.

Reply:
```suggestion
    LoadError::new_raw(interp, message)
```